README.md

```diff
@@ -18,6 +18,13 @@ For details on the JSON Schema used by UBI to send and receive queries and event
 * [Query Response Schema](https://o19s.github.io/ubi/docs/html/query.response.schema.html)
 * [Event Schema](https://o19s.github.io/ubi/docs/html/event.schema.html)
 
+## UBI, Data Prepper, and Open Telemetry
+
+The UBI plugin can store UBI query data in one of two ways:
```

Comment:
I believe we still have the old plugin direct to opensearch end point, no data prepper, no open telementry right?   I think we should change this....

Reply:
No, I don't remember why right off but we took out the direct indexing of UBI events. It has to go through Data Prepper right now (either as JSON or OTel).